src/partials/partial-admin-toolbar.hbs

```diff
@@ -17,13 +17,19 @@
             </ul>
           {{/ inherit }}
           <ul class="list-inline pull-right">
-            {{# inherit stache.config.editInGitHub editInGitHub }}
-              <li>{{{ include stache.config.partial_edit_github }}}</li>
-            {{/ inherit }}
+            {{# if stache.config.editInVSTS }}
```

Comment:
They are mutually exclusive, but I wanted an extra check in there to make sure. In theory, someone could forget to turn off the `editInGitHub` (it's enabled by default) and have both links render. This is just a failsafe.